src/viewport.js

```diff
@@ -198,6 +198,14 @@ export class Viewport {
   }
 
   /**
+   * Returns the scroll height of the content within the viewport.
+   * @return {number}
+   */
+  getScrollHeight() {
+    return this.binding_.getScrollHeight();
```

Comment:
If true, then also true for scrollWidth: Shouldn't we able to pretty aggressively cache this? (Same as size?)